docs/changelog.md

```diff
@@ -1,6 +1,7 @@
 dev
 
 - [docs] Fix the command for [installing development version](https://pypa.github.io/pipx/installation/#install-pipx-development-versions). (#801)
+- Add `list --short` option. (#804)
```

Comment:
Could you briefly mention what this does? i.e. `Add `list --short` option, which only lists package names in list output

Reply:
Done.